data/registry/instrumentation-go-goyek.yml

```diff
@@ -0,0 +1,19 @@
+# cSpell:ignore goyek, otelgoyek, Pająk
+title: goyek instrumentation
```

Comment:
```suggestion
title: Goyek instrumentation
```

Reply:
Even if the library is named "lower case"?

Reply:
Right, I understood that the package ("brand") name is `goyek` but this is a title.

Also, from your logo, it looks like GOyek to me :)
Or maybe it's GOYEK. All caps would make more sense to me.

Regardless, I don't feel strongly about any of this, but my _preference_ would be for the first letter of the title to be capitalized.

Reply:
I respect your preference :wink: Applying suggestion